go.mod

```diff
@@ -3,32 +3,32 @@ module github.com/hashicorp/packer-plugin-azure
 go 1.19
 
 require (
-	github.com/Azure/azure-sdk-for-go v64.0.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.19
-	github.com/Azure/go-autorest/autorest/adal v0.9.20
-	github.com/Azure/go-autorest/autorest/azure/auth v0.4.2
-	github.com/Azure/go-autorest/autorest/azure/cli v0.4.2
-	github.com/Azure/go-autorest/autorest/date v0.3.0
-	github.com/Azure/go-autorest/autorest/to v0.4.0
+	github.com/Azure/go-autorest/autorest v0.11.29
+	github.com/Azure/go-autorest/autorest/adal v0.9.23 // indirect
+	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
 	github.com/approvals/go-approval-tests v0.0.0-20210131072903-38d0b0ec12b1
-	github.com/dnaeon/go-vcr v1.1.0 // indirect
 	github.com/golang-jwt/jwt v3.2.2+incompatible
 	github.com/google/go-cmp v0.5.9
-	github.com/hashicorp/go-azure-helpers v0.16.5
+	github.com/hashicorp/go-azure-helpers v0.56.0
 	github.com/hashicorp/hcl/v2 v2.16.2
-	github.com/hashicorp/packer-plugin-sdk v0.5.1
+	github.com/hashicorp/packer-plugin-sdk v0.4.0
```

Comment:
I had downgraded temporarily this please let me know what version I should upgrade this back to for the release!